### DISCUSSION TITLE
Colour scheme

### DISCUSSION THREAD

**Discussion by ian-tiggeman** [ian-tiggeman/T24S2T1EXTG08#3] [Ideas]
Cara, do you have any ideas or preferences about colours within our calculator app?
I made the main menu a very light purple, but if you like something more fancy, let's chat about it.
Thanks, Ian

**Comment by DanyaoWang-001167576** [comment]
Hi Ian,
The main menu page looks great and the lavender background colour fits perfectly!
Thanks for designing the menu page :)
Cara

**Reply by ian-tiggeman** [reply]
Excellent.  I'm glad you like it.
I have used lavender on my feature page background too.
Regards,
Ian
